Src/GoogleApis.Auth.DotNet4/GoogleApis.Auth.DotNet4.csproj

```diff
@@ -33,6 +33,9 @@
     <DocumentationFile>Google.Apis.Auth.PlatformServices.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="BouncyCastle.Crypto">
+      <HintPath>..\..\packages\BouncyCastle.1.7.0\lib\Net40-Client\BouncyCastle.Crypto.dll</HintPath>

```

Comment:
Just to double check (I'll try to check it myself tomorrow morning...) where is it mentioned that this dll is signed?